ehr_billing/api-src/org/labkey/api/ehr_billing/pipeline/InvoicedItemsProcessingService.java

```diff
@@ -58,4 +67,26 @@ static InvoicedItemsProcessingService get()
      * @param invoiceId invoice Run Id for which additional processing can be performed
      */
     void performAdditionalProcessing(String invoiceId, User user, Container container);
+
+    default Pair<String,String> verifyBillingRunPeriod(User user, Container container, Date startDate, Date endDate) throws PipelineJobException
+    {
+        // first look for existing records overlapping the provided date range.
+        // so this should not be a problem
```

Comment:
This comment seems strange. "Should not be a problem" because we're going to explicitly disallow it? Or the thing we're checking for should never happen?